journal-of-computer-applications-in-archaeology.csl

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Journal of Computer Applications in Archaeology</title>
+    <title-short>JCAA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-computer-applications-in-archaeology</id>
+    <link href="http://www.zotero.org/styles/journal-of-computer-applications-in-archaeology" rel="self"/>
+    <link href="http://www.zotero.org/styles/citizen-science-theory-and-practice" rel="template"/>
+    <link href="http://journal.caa-international.org/about/submissions/#References" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <eissn>2514-8362</eissn>
+    <summary>Harvard author-date style edited for Journal of Computer Applications in Archaeology.</summary>
+    <updated>2017-05-24T15:41:19+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="accessed">last accessed</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" initialize-with=" " name-as-sort-order="all"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <text macro="editor"/>
+        <choose>
+          <if type="report" match="any">
+            <text variable="publisher"/>
+          </if>
+        </choose>
```

Comment:
@damnation333, I assume it's better to add the `<names variable="translator"/>` here as well. It should always be possible to match the in-text citation against the bibliographic entry, so the author-macros almost always need the exact same substitution logic.